xchange-examples/src/main/java/org/knowm/xchange/examples/btcchina/account/BTCChinaAccountDemo.java

```diff
@@ -52,6 +55,16 @@ public static void generic() throws IOException {
     // API key has no withdraw rights - returns 401 unauthorized
     // String withdrawResult = accountService.withdrawFunds(new BigDecimal(1).movePointLeft(5), "1CoPAWJtran45gNM21te1xgZqbDd5UqYWB");
     // System.out.println("withdrawResult = " + withdrawResult);
+
+    fundsInfo(accountService);
+  }
+
+  private static void fundsInfo(AccountService accountService) throws IOException {

```

Comment:
This should be made generic, i.e. no references to BTC China. See [this comment](https://github.com/timmolter/XChange/pull/1406#issuecomment-288047134).